package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "discourse-fingerprint",
+  "version": "1.0.0",
```

Comment:
It's 2.0! 😅 https://github.com/discourse/discourse-fingerprint/blob/c91883fd2cfd46c266ed0592abdbed2598dc8c6d/plugin.rb#L5